flask/boxwise_flask/models/box.py

```diff
@@ -92,5 +92,8 @@ def create_box(box_creation_input):
 
     @staticmethod
     def get_box(box_id):
-        box = Box.select().where(Box.box_id == box_id).get()
-        return box
+        return Box.select().where(Box.box_id == box_id).get()
```

Comment:
It should be possible to use the shorter `return Box.get(Box.box_id == box_id)`, see [here](http://docs.peewee-orm.com/en/latest/peewee/querying.html#selecting-a-single-record); also below

Reply:
Nice catch--thanks!